jsapp/js/components/modalForms/dataAttachmentColumnsForm.es6

```diff
@@ -0,0 +1,226 @@
+      t('Import data from ##PARENT_NAME##')
+        .replace('##PARENT_NAME##', this.props.parent.name)
+    );
+  }
+
+  onAttachToParentCompleted() {
+    this.props.onModalClose();
+  }
+  onAttachToParentFailed() {
+    this.setState({isLoading: false});
+  }
+  onBulkSelect() {
+    let newList = this.state.columnsToDisplay.map((item) => {
+      return {label: item.label, checked: true}
+    });
+    this.setState({columnsToDisplay: newList})
+  }
+  onBulkDeselect() {
+    let newList = this.state.columnsToDisplay.map((item) => {
+      return {label: item.label, checked: false}
+    });
+    this.setState({columnsToDisplay: newList})
+  }
+  onLoadAssetContentCompleted(response) {
+    if (
+      response.data_sharing?.fields.length > 0
+    ) {
+      this.setState({
+        isVirgin: false,
+        columnsToDisplay: this.props.generateColumnFilters(
+          this.props.fields,
+          response.data_sharing.fields,
+        ),
+      });
+    } else {
+      // empty `fields` implies all parent questions are exposed
+      this.setState({
+        isVirgin: false,
+        columnsToDisplay: this.props.generateColumnFilters(
+          this.props.fields,
+          response.content.survey,
+        ),
+      });
+    }
+  }
+  onPatchParentCompleted(response) {
+    this.setState({
+      isLoading: false,
+      columnsToDisplay: this.props.generateColumnFilters(
+        this.props.fields,
+        response.fields,
+      ),
+    });
+    this.props.onModalClose();
+  }
+  onPatchParentFailed() {
+    this.setState({isLoading: false});
+  }
+
+  onColumnSelected(newList) {
+    this.setState({columnsToDisplay: newList});
+  }
+
+  onSubmit(evt) {
+    evt.preventDefault();
+    this.setState({isLoading: true});
+    this.props.triggerParentLoading();
+
+    let fields = [];
+    var data = '';
+
+    this.state.columnsToDisplay.map((item) => {
+      if (item.checked) {
+        fields.push(item.label);
+      }
+    });
+
+    if (this.props.attachmentUrl) {
+      data = JSON.stringify({
+        fields: fields,
+        filename: this.props.filename,
+      });
+      actions.dataShare.patchParent(this.props.attachmentUrl, data);
+    } else {
+      data = JSON.stringify({
+        parent: this.props.parent.url,
+        fields: fields,
+        filename: this.props.filename,
+      });
+      actions.dataShare.attachToParent(this.props.asset.uid, data);
+    }
+  }
+
+  render() {
+    return (
+      <bem.FormModal__form m='data-attachment-columns'>
+        <div className='header'>
+          <span className='modal-description'>
+            {t('You are about to import ##PARENT_NAME##. Select or deselect in the list below to narrow down the number of questions to import.').replace('##PARENT_NAME##', this.props.parent.name)}
+          </span>
+
+          <div className='bulk-options'>
+            <span className='bulk-options__description'>
+              {t('Select below the questions you want to import')}
```

Comment:
Maybe rather one of these?:
1. Select the questions you want to import;
2. Select the questions you want to import from the list below;
3. Select the questions you want to import from below;
4. Select which questions to import;
5. Select which questions to import from below;

Reply:
I copied the text you see from the mock ups, maybe @Ig-Rebollo can comment?